cmake/onnxruntime_providers.cmake

```diff
@@ -732,9 +733,9 @@ if (onnxruntime_USE_OPENVINO)
 
   if ((DEFINED ENV{OPENCL_LIBS}) AND (DEFINED ENV{OPENCL_INCS}))
     add_definitions(-DIO_BUFFER_ENABLED=1)
-    list(APPEND OPENVINO_LIB_LIST $ENV{OPENCL_LIBS} ${InferenceEngine_LIBRARIES} ${NGRAPH_LIBRARIES} ngraph::onnx_importer ${PYTHON_LIBRARIES})
+    list(APPEND OPENVINO_LIB_LIST $ENV{OPENCL_LIBS} openvino::frontend::onnx openvino::runtime ${InferenceEngine_LIBRARIES} ${NGRAPH_LIBRARIES} ngraph::onnx_importer ${PYTHON_LIBRARIES})
   else()
-    list(APPEND OPENVINO_LIB_LIST ${InferenceEngine_LIBRARIES} ${NGRAPH_LIBRARIES} ngraph::onnx_importer ${PYTHON_LIBRARIES})
+    list(APPEND OPENVINO_LIB_LIST openvino::frontend::onnx openvino::runtime ${InferenceEngine_LIBRARIES} ${NGRAPH_LIBRARIES} ngraph::onnx_importer ${PYTHON_LIBRARIES})
```

Comment:
with 2022.1, the ${NGRAPH_LIBRARIES} ngraph::onnx_importer won't be part of OPENVINO_LIB_LIST. should we try to add different checks for openvino version and only add the appropriate dependencies else it is fine anyways?

Reply:
they are needed to maintain backward compatibility with 2021.4

Reply:
Right. I was thinking to have some cool cmake checks like:

if (OPENVINO_VERSION VERSION_LESS_EQUAL "2020.4")
    list(APPEND OPENVINO_LIB_LIST ${InferenceEngine_LIBRARIES} ${NGRAPH_LIBRARIES} ngraph::onnx_importer 
else()
   list(APPEND OPENVINO_LIB_LIST openvino::frontend::onnx openvino::runtime ${InferenceEngine_LIBRARIES} ${NGRAPH_LIBRARIES} ngraph::onnx_importer ${PYTHON_LIBRARIES})
  endif()

Reply:
But I think whatever we have should be fine.